protocol/vm/vm_test.go

```diff
@@ -460,10 +416,18 @@ func TestVerifyTxInputQuickCheck(t *testing.T) {
 				ok = false
 			}
 		}()
-		tx := bc.NewTx(bc.TxData{
-			Inputs: []*bc.TxInput{bc.NewSpendInput(witnesses, bc.Hash{}, bc.AssetID{}, 10, 0, program, bc.Hash{}, nil)},
-		})
-		verifyTx(tx, 0)
+
+		vctx := &Context{
+
+			VMVersion: 1,
+			Code:      program,
+			Arguments: witnesses,
+			MinTimeMS: new(uint64),
+			MaxTimeMS: new(uint64),
+			TxData:    &[]uint8{0xa7, 0xff, 0xc6, 0xf8, 0xbf, 0x1e, 0xd7, 0x66, 0x51, 0xc1, 0x47, 0x56, 0xa0, 0x61, 0xd6, 0x62, 0xf5, 0x80, 0xff, 0x4d, 0xe4, 0x3b, 0x49, 0xfa, 0x82, 0xd8, 0xa, 0x4b, 0x80, 0xf8, 0x43, 0x4a},
```

Comment:
Can you add a comment explaining the significance of these numbers (or lack thereof)?